modules/customize-certificates-api-add-named.adoc

```diff
@@ -18,6 +18,17 @@ client's URL.
 reach the API server.
 * The certificate must have the `subjectAltName` extension for the URL.
 
+If a certificate chain is used, all certificates must be manually concatenated into a single named certificate file. This can be achieved with the following example: `cat first_cert.pem second_cert.pem third_cert.pem >combined_cert.pem`.
+
+The order of the chain should be:
+
+* {product-title} master host certificate.
+
+* Intermediate CA certificate.
+
+* Root CA certificate.
```

Comment:
```suggestion
If a certificate chain is required to certify the server certificate, then the certificate chain must be appended to the server certificate. For example: 

----
$ cat server_cert.pem int2ca_cert.pem int1ca_cert.pem rootca_cert.pem>combined_cert.pem.
----

When combining certificates, the order of the certificates is important. Each following certificate must directly certify the certificate preceding it, for example:

1. {product-title} master host server certificate.

2. Intermediate CA certificate that certifies the server certificate.

3. Root CA certificate that certifies the intermediate CA certificate.
```

Reply:
Thanks @sanchezl - I've incorporated your suggested changes. Other than the numbering formatting, the only other thing I changed was to remove a period at the end of your `cat` command after `combined_cert.pem`. Please let me know if that period was required for some reason. Thanks!

Reply:
@bergerhoffer , A supplement to the certificate extension，added just like below,

If a certificate chain is required to certify the server certificate, then the certificate chain must be appended to the server certificate. Certificate files must be Base64 PEM-encoded and typically have a .crt or .pem extension. For example
...

Reply:
Thanks @wangke19 - I'll add the additional sentence!